src/Analyzer.JsonRuleEngine/JsonRuleEngine.cs

```diff
@@ -64,7 +64,8 @@ public void FilterRules(ConfigurationDefinition configuration)
             if (configuration == null)
                 throw new ArgumentNullException(nameof(configuration));
 
-            if (configuration.InclusionsConfigurationDefinition != null)
+            if (configuration.InclusionsConfigurationDefinition?.Severity != null
```

Comment:
Should you make this check for exclusions as well?

Reply:
I think I originally left it out because since only the inclusions object overrides the exclusions object, the exclusions object wouldn't cause the same issue if it was empty (but not null) because it doesn't affect overriding anything. Regardless I just added some more tests for exclusions listed first and such just for more tests, defensive coding, etc.